source/crud/tests/v2/default-write-concern.yml

```diff
@@ -0,0 +1,413 @@
+            - name: insertOne
+              arguments:
+                document:
+                  _id: 1
+            - name: updateOne
+              arguments:
+                filter:
+                  _id: 1
+                update:
+                  $set:
+                    x: 1
+            - name: insertOne
+              arguments:
+                document:
+                  _id: 2
+            - name: replaceOne
+              arguments:
+                filter:
+                  _id: 1
+                replacement:
+                  x: 2
+            - name: insertOne
+              arguments:
+                document:
+                  _id: 3
+            - name: updateMany
+              arguments:
+                filter:
+                  _id: 1
+                update:
+                  $set:
+                    x: 3
+            - name: deleteOne
+              arguments:
+                filter:
+                  _id: 3
+    outcome:
+      collection:
+        name: default_write_concern_coll
+        data:
+          - _id: 1
+            x: 3
+          - _id: 2
+    expectations:
+      - command_started_event:
+          command:
+            delete: default_write_concern_coll
+            deletes:
+              - q: {}
+                limit: 0
+            writeConcern: null
+      - command_started_event:
+          command:
+            insert: default_write_concern_coll
+            documents:
+              - _id: 1
+            writeConcern: null
+      - command_started_event:
+          command:
+            update: default_write_concern_coll
+            updates:
+              - q:
+                  _id: 1
+                u:
+                  $set:
+                    x: 1
+            writeConcern: null
+      - command_started_event:
+          command:
+            insert: default_write_concern_coll
+            documents:
+              - _id: 2
+            writeConcern: null
+      - command_started_event:
+          command:
+            update: default_write_concern_coll
+            updates:
+              - q:
+                  _id: 1
+                u:
+                  x: 2
+            writeConcern: null
+      - command_started_event:
+          command:
+            insert: default_write_concern_coll
+            documents:
+              - _id: 3
+            writeConcern: null
+      - command_started_event:
+          command:
+            update: default_write_concern_coll
+            updates:
+              - q:
+                  _id: 1
+                u:
+                  $set:
+                    x: 3
+                multi: true
+            writeConcern: null
+      - command_started_event:
+          command:
+            delete: default_write_concern_coll
+            deletes:
+              - q:
+                  _id: 3
+                limit: 1
+            writeConcern: null
+  - description: 'findOneAnd{update,delete,replace}'
```

Comment:
"findAndModify operations omit default write concern".

Note that this test would require MongoDB 3.2+ (per https://github.com/mongodb/specifications/blob/master/source/wireversion-featurelist.rst).

Reply:
Done.